packages/search-ui/src/__tests__/actions/setSearchTerm.test.js

```diff
@@ -100,20 +126,19 @@ describe("#setSearchTerm", () => {
     expect(subject("term", { refresh: false }).state.wasSearched).toBe(false);
   });
 
-  it("Will not debounce requests if there is no debounce specified", async () => {
+  it("Will not debounce requests if there is no debounce specified", () => {
     const { driver, mockApiConnector } = setupDriver();
     driver.setSearchTerm("term", { refresh: true });
-    driver.setSearchTerm("term", { refresh: true });
-    driver.setSearchTerm("term", { refresh: true });
```

Comment:
Don't we still want to keep these 2 other calls to make sure that it's not debouncing (length = 3)? Or am I missing something with this test?

Reply:
Fixed in https://github.com/elastic/search-ui/pull/391/commits/159f8c1e6b5afcda13b6ee29acc8fbc946c16d40 with an explanation.

Reply:
Awesome. Thanks for explaining that - this looks great!